docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/generic.md

```diff
@@ -196,21 +234,28 @@ actions:
 resources that contain subscriptions for a provided repository URL.
 :::
 
-### Expression functions
+### Expression Reference
 
 The Generic webhook receiver extends
 [built-in expr-lang support](https://expr-lang.org/docs/language-definition) 
-with utilities that can be used to help derive `TargetSelectionCriteria` 
-information from incoming requests.
-
-The following expression functions are available:
+with utilities that can be used to help derive `targetSelectionCriteria` 
+information from incoming requests. The following reference contains the
+variables and functions available for yeilding expression derived values.
+
+- [request.body](#requestbody)
+- [request.header](#requestheaderheaderkey)
+- [request.headers](#requestheadersheaderkey)
+- [request.params](#requestparamsqueryparamkey)
+- [normalizeGit](#normalizegiturl)
+- [normalizeImage](#normalizeimageurl)
+- [normalizeChart](#normalizecharturl)
 
 #### request.body
 
-Derived from incoming requests whose fields can be accessed using
-standard bracket or dot-notation. For example, `data.address.city` would access 
-the `city` property nested within the `address` object, and `data.users[0]` 
-would access the first item in a `users` array. 
+Derived from incoming requests whose fields can be accessed using bracket or 
+dot-notation. For example, `data.address.city` would access the `city` property 
+nested within the `address` object, and `data.users[0]` would access the first 
+item in a `users` array.
```

Comment:
"accessed using bracket or dot notation" is nicely instructive, and I'd keep it, but I'd lead with the more technical statement that `request.body` is _structured data_ parsed from the original request body which (this is important) is assumed to contain valid JSON.